sechub-doc/src/docs/asciidoc/sechub-getting-started/run-all-steps.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Get sechub's root folder
+currentDir=$(pwd)
+baseDir="$currentDir"
+regex=sechub$
+
+while ! [[ "$baseDir" =~ ${regex} ]]
+do
+baseDir="$(dirname $baseDir)"
+done
+
+
+# Copy all scripts from the current folder
+cp ./* "$baseDir"
+rm "$baseDir/run-all-steps.sh"
+cd $baseDir
+
+sechub_pid=0
+pds_pid=0
+
+stopServers(){
+    kill "$sechub_pid"
+    kill "$pds_pid"
+    exit 0
+}
+
+# Start SecHub
+$(sh ./start-sechub.sh) &
```

Comment:
The better approach would be to check whether the server is alive or not. Have a look at the `test-tern.sh` and `run-suite.sh` in the sechub/sechub-pds-solutions/tern/tests folder for inspiration.

However, that might just be a bit of an overkill.